server/chat-commands/admin.ts

```diff
@@ -1102,3 +1121,14 @@ export const commands: ChatCommands = {
 		`[player] must be a username or number, [pokemon] must be species name or party slot number (not nickname), [move] must be move name.`,
 	],
 };
+
+export const pages: PageTable = {
+	bot(args, user) {
+		const [bot, pageid] = args;
+		const targetBot = Users.get(bot);
+		if (!targetBot) {
+			return `<div class="pad"><h2>User ${bot} not found, and so cannot be requested for this page.</h2></div>`;
```

Comment:
```suggestion
			return `<div class="pad"><h2>The bot "${bot}" is not available.</h2></div>`;
```

(Maybe have a bot check so it can't be used to spam regular users?)